media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -0,0 +1,217 @@
+    global default_user_json
+    tools.argparser.add_argument('--user-credentials',
+                                 default=default_user_json,
+                                 help='Filename containing Google user credentials')
+
+    global verbose
+    tools.argparser.add_argument('--verbose',
+                                 action='store_true',
+                                 default=verbose,
+                                 help='If enabled, emit extra status messages during run')
+    global debug
+    tools.argparser.add_argument('--debug',
+                                 action='store_true',
+                                 default=debug,
+                                 help='If enabled, emit even more extra status messages during run')
+    global logfile
+    tools.argparser.add_argument('--logfile',
+                                 default=logfile,
+                                 help='Store verbose/debug logging to the specified file')
+
+    tools.argparser.add_argument('--donothing',
+                                 action='store_true',
+                                 help='....')
+           
+
+    global args
+    args = tools.argparser.parse_args()
+
+    # --debug implies --verbose
+    if args.debug:
+        args.verbose = True
+
+    return args
+
+####################################################################
+
+def check_response_status(events, calendar, log):
+    #iterate through the list of events and find ones where responseStatus = needsAction
+
+    needs_action_events = []
+    for event in events:
+        if "attendees" in event: #determines whether the event has any attendees
+            for attendee in event["attendees"]:
+                if attendee["email"] == calendar['id']:
+                    if attendee["responseStatus"] == "needsAction":
+                        needs_action_events.append(event)
+
+    log.info(f"Found {len(needs_action_events)} needsAction events from {calendar['name']} {calendar['id']}")
+
+    return needs_action_events
+
+####################################################################
+
+def respond_to_events(events, service, calendar, log):
+    #accepts events from a domain in the list of acceptable domains, declines all others
+
+    for event in events:
+        organizer_email = event["organizer"]["email"]
+        organizer_domain = organizer_email.split('@')[1]
+        id = event["id"]
+ 
+        if organizer_domain in acceptable_domains:
+            response = 'accepted'
+        else:
+            response = 'declined'
+
+        log.info(f"Event {event['summary']} {id}: will be {response}")
+
+        response_body =   {
+                "attendees" : [
+                    {
+                        "email" : calendar['id'],
+                        "responseStatus" : response,
+                    },
+                ],
+            }
+
+        if not args.donothing:
```

Comment:
If you change the option name to `--dry-run`, this will change to `args.dry_run`.